Storage/src/SigningHelper.php

```diff
@@ -882,4 +896,47 @@ private function buildQueryString(array $input)
 
         return implode('&', $q);
     }
+
+    /**
+     *   Retry logic for signBlob using RetryTrait.
```

Comment:
nit - remove extra leading whitespace
```suggestion
     * Retry logic for signBlob
```

Reply:
Fixed